static/update.sh

```diff
@@ -30,6 +30,8 @@ then
 fi
 
 # Delete, download, run
-run_script GITHUB_REPO nextcloud_update
+run_script GITHUB_REPO nextcloud_update || exit 1
+
+#placeholder_for_automatic_updates
 
 exit
```

Comment:
dunno if this is needed?

Reply:
yah, it might not be needed but is the current placeholder. I don't think that introduction a placeholder string is the better approach.